index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="desk.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <nav id="navbar">
+        <ul id="list">
+            <li><a id="home" href="index.html">Home</a></li>
+            <li><a id="portfolio" href="portfolio.html">Portfolio</a></li>
+            <li><a id="about" href="about.html">About</a></li>
+            <li><a id="contact" href="contact.html">Contact</a></li>
+            <li><img id="line" src="./images/Rectangle_52.png" alt="rectangle to underline options"></li>
+        </ul>
+    </nav>
+
+    <main>
+        <section id="presentation">
+            <article>
+                <p id="name">Francisco Borja</p>
+                <p id="phrase">Hello! I am a software developer! <wbr>I can help you build a product, <br>feature or
+                    website. <wbr>Take a look of my works.<br>
+                    If you like what you see and have a project <wbr>you need coded, <br>don’t hesitate and contact me.
+                </p>
+            </article>
+            <div>
+                <p id="txtbutton">Start collaboration</p>
+            </div>
+        </section>
+        <aside id="images">
+            <img id="geometry" src="./images/image-geometry_1.png" alt="colorfull geometry design">
+            <img id="dots" src="./images/image_dots.png" alt="dots">
+        </aside>
+    </main>
```

Comment:
- [ ] Please the list of icons uses a `<ul>` tag to group them, and `<li>` per icon; each icon has a link (`<a>` tag) 👍

Reply:
not resolved